### DISCUSSION TITLE
Table - setting filter after initialization

### DISCUSSION THREAD

**Discussion by jzbuchalski** [iTwin/iTwinUI#943] [General]
Is it possible to explicitly set a filter on the Table after initialization?  We have manualFilters set to **true**.  In some cases, we want to manually filter the table (contents) and have the filter icon appear.  Filtering the table is working for us fine, but how can we update the filter icon to appear on given columns?  Setting initialState.filters have no effect since the table has already been initialized.  I tried messing with stateReducer, but have been unsuccessful.  Any suggestions?

Sorry, never mind, found a solution :-)

**Comment by mayank99** [comment]
> Sorry, never mind, found a solution :-)

@jzbuchalski Can you post your solution here? It could help others who might stumble upon this discussion in the future.
